kiota_authentication_azure/azure_identity_access_token_provider.py

```diff
@@ -45,10 +47,14 @@ def __init__(
         self._credentials = credentials
         self._scopes = scopes
         self._options = options
+        self._is_cae_enabled = is_cae_enabled
         self._allowed_hosts_validator = AllowedHostsValidator(allowed_hosts)
 
     async def get_authorization_token(
-        self, uri: str, additional_authentication_context: Dict[str, Any] = {}
+        self,
+        uri: str,
+        additional_authentication_context: Dict[str, Any] = {},
+        is_cae_enabled: bool = True
```

Comment:
adding a parameter to this method deviates from the interface contract. We should remove it and rely on the property/field only instead.

Reply:
I have removed it altogether, and relied only on the value from the constructor

Reply:
thanks, resolving